src/components/Logo.js

```diff
@@ -2,41 +2,17 @@ import { Box } from '@mui/material';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import CirclesLogoGroupSVG from '%/images/logo-group.svg';
 import CirclesLogoSVG from '%/images/logo.svg';
 
-const Logo = ({ isWithGang, size = 'default' }) => {
+const Logo = ({ size = 'default' }) => {
   const sizes = {
```

Comment:
This constant could be extracted and declared outside the functional component. There is not need to let it within the function to be redeclared everytime the component is instantiated since it does not depend from component props and is just sizes definition.

Reply:
it actually depends on line 18 and 20:
`height={sizes[size]}`
`width={sizes[size]}`

Reply:
I meant that the variable's value does not depend on anything. The `sizes` object is a constant always with the same value so can be declared outside the `Logo` component.

Reply:
ok, I got it now :). Thx I corrected it :)